src/Symfony/Component/PropertyInfo/PropertyInfo.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * Default {@see PropertyInfoInterface} implementation.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PropertyInfo implements PropertyInfoInterface
+{
+    /**
+     * @var PropertyListRetrieverInterface[]
+     */
+    private $listExtractors;
+    /**
+     * @var PropertyTypeInfoInterface[]
+     */
+    private $typeExtractors;
+    /**
+     * @var PropertyDescriptionInfoInterface[]
+     */
+    private $descriptionExtractors;
+    /**
+     * @var PropertyAccessInfoInterface[]
+     */
+    private $accessExtractors;
+
+    /**
+     * @param PropertyListRetrieverInterface[]   $listExtractors
+     * @param PropertyTypeInfoInterface[]        $typeExtractors
+     * @param PropertyDescriptionInfoInterface[] $descriptionExtractors
+     * @param PropertyAccessInfoInterface[]      $accessExtractors
+     */
+    public function __construct(array $listExtractors = array(), array $typeExtractors = array(),  array $descriptionExtractors = array(), array $accessExtractors = array())
```

Comment:
CS seems to be fine, but IMO it is more readable, if we use line breaks after each parameter